protostar/starknet_gateway/gateway_facade.py

```diff
@@ -446,6 +449,33 @@ async def _create_contract_function(
         except KeyError:
             raise UnknownFunctionException(function_name) from KeyError
 
+    async def _get_resolved_contract_function(
```

Comment:
```suggestion
    async def _contract_from_possibly_proxy_address(
```

Reply:
Methods should start with a verb. Renaming to `_get_contract_from_possibly_proxy_address`.